19 спринт

alk8 opened the pull request:
Переверстано again))))))))

Commit:
19 Рефакторинг на фраггменты

Commit:
19 Верстка

Commit:
19 Верстка удалены старые обьекты

Commit:
19 Верстка Скрытие клавиатуры

nadzyathedude commented on app/src/main/java/com/example/playlistmaker/presentation/player/PlayerFragment.kt:
```diff
@@ -0,0 +1,125 @@
+
+        timer = binding.timer
+        play = binding.playButton
+        isDark = isDarkTheme()
```
можно не объявлять, а напрямую в нужных местах обращаться через binding

nadzyathedude commented on app/src/main/java/com/example/playlistmaker/presentation/player/PlayerFragment.kt:
```diff
@@ -0,0 +1,125 @@
+            parametersOf(requireArguments().getString(TRACK), toast)
+        }
+
+        timer = binding.timer
+        play = binding.playButton
```
можно не объявлять, а напрямую в нужных местах обращаться через binding

nadzyathedude commented on app/src/main/java/com/example/playlistmaker/presentation/search/SearchFragment.kt:
```diff
@@ -0,0 +1,216 @@
+        parametersOf(runnable)
+    }
+
+    @SuppressLint("NotifyDataSetChanged")
```
можно убрать аннотацию, она ни на что не влияет. Студия подсветила `notifyDataSetChanged() `потому что этот метод замедляет работу recyclerView и рекомендуется использовать другие методы, но это не точно не критично

nadzyathedude commented on app/src/main/java/com/example/playlistmaker/presentation/settings/SettingsFragment.kt:
```diff
@@ -0,0 +1,71 @@
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentSettingsBinding.inflate(inflater,container,false)
```
обычно префикс _  используется, когда переменна приватная

Commit:
19 Верстка

С экрана плеера невозможно вернутся обратно на экран поиска, панели навигации не должно быть на экране плеера
Элемент «Медиатека» является выбранным по умолчанию - сейчас первым открывается экран поиска
